src/Header.svelte

```diff
@@ -170,7 +170,7 @@
     {:else}
       <span class="network unavailable">No Network</span>
     {/if}
-    <a use:link class="register" href="/registrations">Register</a>
+    <a use:link class="register" href="/registration">Register</a>
```

Comment:
Why is this being changed? It's inconsistent with the other resources that are list-like, and it's a breaking change.

Reply:
This was my doing. My thinking behind it was that most of our routes are singular and we only have one `registration` screen. The other screens being:
```
/faucet
/vesting
```
So I wanted to align that. This also would better align with routes that actually distinguish between singular and plural, like `commit` vs `commits` which are two different sub-routes/screens of the project screen.

Reply:
the plural routes are the ones that represent more than one sub-resources, eg. orgs, projects, seeds, registrations

Reply:
fixed in 1232ca3b3595029730d172920de6c48057927a61